blocks/links-bar-block/features/links-bar/default.jsx

```diff
@@ -35,19 +34,23 @@ const LinksBar = ({ customFields: { navigationConfig = {}, ariaLabel } }) => {
 		<>
 			<nav
 				key={id}
-				className="links-bar"
+				className={BLOCK_CLASS_NAME}
 				aria-label={ariaLabel || phrases.t("links-bar-block.element-aria-label")}
 			>
 				{menuItems &&
 					menuItems.map((item, index) => (
-						<PrimaryFont as="span" className="links-menu" key={item._id}>
+						<span className={`${BLOCK_CLASS_NAME}-menu`} key={item.display_name}>
```

Comment:
can you use the bem naming standard of using "__" rather than "-" for an element of a block http://getbem.com/naming/
```suggestion
						<span className={`${BLOCK_CLASS_NAME}__menu`} key={item.display_name}>
```